### DISCUSSION TITLE
Best way to "reupload" all images in a collection to trigger hooks and new sizes?

### DISCUSSION THREAD

**Discussion by christian-reichart** [payloadcms/payload#1934] [Q&A] [answered]
I've added new sizes, an "filename renaming" beforeOperation hook and a "base64 placeholder string" beforeChange hook to one of my upload collections.

What's the best way to "recreate" all the existing files to generate new images, a new filename and the base64 placeholder?

Thanks for any hints!

**Comment by jacobsfletch** [comment, marked as answer]
@christian-reichart check out https://github.com/payloadcms/payload/discussions/1834

**Reply by christian-reichart** [reply]
Oh wow thank you, will try this soon!
